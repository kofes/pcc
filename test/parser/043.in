var
	arr : array[13..29,7..11] of integer;
begin
end
