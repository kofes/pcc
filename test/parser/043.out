<function table>
</function table>
<type table>
   <type>BOOLEAN</type>
   <type>CHAR</type>
   <type>INTEGER</type>
   <type>REAL</type>
</type table>
<var table>
   <var>ARR:V:ARRAY[13:29] OF ARRAY[7:11] OF INTEGER</var>
   <var>PI:C:REAL:3.1415926535897932385</var>
</var table>
<block statement>
</block statement>
